src/pages/home/ReportScreen.js

```diff
@@ -151,6 +152,9 @@ class ReportScreen extends React.Component {
             Report.handleInaccessibleReport();
             return;
         }
+
+        // Always reset the state of the composer view when the current reportID changes
+        toggleReportActionComposeView(true);
```

Comment:
Not following. What is not ideal about it?

Reply:
Assuming users on web & desktop have small screens (in the code) seems not ideal since I assume most people are on large screens. Not a blocker at all, just wondering what your thoughts were on this, can resolve this here